core/thread.py

```diff
@@ -336,6 +343,72 @@ def _close_after(self, closer, silent, delete_channel, message):
                 if str(message_id) == str(embed.author.url).split("/")[-1]:
                     return msg
 
+    async def _grab_timeout(
+            self
+    ) -> typing.Union[None, isodate.duration.Duration, timedelta]:
+        """
+        This grabs the timeout value for closing threads automatically
+        from the ConfigManager and parses it for use internally.
+
+        :returns: None if no timeout is set.
+        """
+        timeout = self.bot.config.get("thread_auto_close")
+        if timeout is None:
+            return timeout
+        else:
+            try:
+                timeout = isodate.parse_duration(timeout)
+            except isodate.ISO8601Error:
+                logger.warning(
+                    "The auto_close_thread limit needs to be a "
+                    "ISO-8601 duration formatted duration string "
+                    'greater than 0 days, not "%s".',
+                    str(timeout),
+                )
+                del self.bot.config.cache["thread_auto_close"]
+                await self.bot.config.update()
+                timeout = None
+        return timeout
+
+    async def _restart_close_timer(self):
+        """
+        This will create or restart a timer to automatically close this
+        thread.
+        """
+        timeout = await self._grab_timeout()
+
+        # Exit if timeout was not set
+        if timeout is None:
+            return
+
+        # Set timeout seconds
+        seconds = timeout.total_seconds()
+        # seconds = 20  # Uncomment to debug with just 20 seconds
+        reset_time = datetime.utcnow() + timedelta(seconds=seconds)
+        human_time = human_timedelta(dt=reset_time)
+
+        # Grab message
+        close_message = self.bot.config.get(
+            "thread_auto_close_response",
+            f"This thread has been closed automatically after no response from"
+            f" you for {human_time}."
```

Comment:
Use this message instead `This thread has been closed automatically due to inactivity after {human_time}`

Reply:
I'll push that commit when I get back home. Thanks!